src/stories/Forms/Checkbox/index.tsx

```diff
@@ -112,10 +117,7 @@ export const Checkbox = React.forwardRef(
           type='checkbox'
           {...additionalInputProps}
         />
-
-        <label htmlFor={id}>
-          <span>{children}</span>
-        </label>
+        <label htmlFor={id}>{children && <span>{children}</span>}</label>
```

Comment:
The has label prop is confusing me a bit when there is a label inside it already? Should maybe the label be wrapped in the check as well?